configure.py

```diff
@@ -131,13 +130,16 @@ def cygpath(path):
   return run_shell('cygpath  -m "%s"' % path)
 
 
-def get_python_path(environ_cp):
+def get_python_path(environ_cp, python_bin_path):
   """Get the python site package paths."""
   python_paths = []
   if environ_cp.get('PYTHONPATH'):
     python_paths = environ_cp.get('PYTHONPATH').split(':')
   try:
-    library_paths = site.getsitepackages()
+    check_input = [python_bin_path, '-c',
+                   'import site; print("\\n".join(site.getsitepackages()))']
+    library_paths = subprocess.check_output(
+        check_input).decode('UTF-8').strip().split("\n")
   except AttributeError:
```

Comment:
Sorry missed this one earlier.  Need to update to "subprocess.CalledProcessError" now that we are calling check_output.